config/locales/en.yml

```diff
@@ -106,57 +106,73 @@ en:
         <li>the Ministry of Housing, Communities & Local Government</li>
         <li>the Department for Environment Food & Rural Affairs</li>
         <li>the Department for Work & Pensions</li>
         <li>the National Health Service (NHS)</li>
       </ul>
-      <p class="govuk-body">We’ll use the personal information you provide through this service to help make sure that people with a medical condition that makes them extremely vulnerable to coronavirus get the support they need.</p>
-      <p class="govuk-body">We’ll use it to check your needs against data provided to us by the NHS, and to contact you with further advice or information.</p>
-      <p class="govuk-body">We’ll also use it to get in touch with you so we can get your feedback on the service and support you have received.</p>
-      <p class="govuk-body">This privacy notice covers:</p>
+      <p class="govuk-body">The service is no longer open for new registrations because people aren’t being advised to shield in most of England from 1 August 2020.</p>
+      <p class="govuk-body">This means we are no longer collecting personal information through the online service or the automated telephone service. But we still receive personal information about individuals who are clinically extremely vulnerable to coronavirus from the NHS.</p>
+      <p class="govuk-body">This page explains what personal information we’re still getting and how we’re protecting the personal information we’ve already collected.</p>
+      <h2 class="govuk-heading-m">Personal information</h2>      
+      <p class="govuk-body">The NHS is still providing us with personal information about individuals who are clinically extremely vulnerable to coronavirus:</p>
       <ul class="govuk-list govuk-list--bullet">
-        <li>the online service, where you can enter the information that we need to get support to you where you live</li>
-        <li>the automated telephone service, where you can provide the same information over the phone</li>
-        <li>online surveys and other tools we may use to get your feedback on the service</li>
+        <li>name</li>
+        <li>address</li>
+        <li>date of birth</li>
+        <li>phone numbers</li>
+        <li>NHS number</li>
       </ul>
+      <p class="govuk-body">We will retain the information we’ve already collected through the online service and automated telephone system:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>name</li>
+        <li>address</li>
+        <li>date of birth</li>
+        <li>email address</li>
+        <li>phone numbers</li>
+        <li>NHS number</li>
+        <li>whether the person needs help getting food and essential supplies</li>
+        <li>whether the person needs help meeting their basic care needs</li>
+        <li>whether the person has special dietary requirements</li>
+        <li>Nwhether there’s someone in the person’s household who can carry a box of supplies inside</li>        
```

Comment:
don't think this N is meant to be there

Reply:
Sorry, yes - it's a typo. Is it easier for you to fix it, or would you rather I did it?

Reply:
I'll fix it now. There also seems to be something that's caused this not to work but I'm not sure what it is yet